package.json

```diff
@@ -23,16 +22,6 @@
     "server-int": "NODE_ENV=test mocha -t 10000 --require ./server/test/helper.js ./server/test/integration --recursive --reporter=nyan --forbid-pending",
     "server-lint": "eslint server --env mocha,chai --global expect,sinon"
   },
-  "lint-staged": {
-    "*.vue": [
-      "eslint --fix --ext .js,.vue",
```

Comment:
I thought you meant you were just going to remove `--fix`, not the eslint check altogether...